plugin/src/test/testProject/settings.gradle.kts

```diff
@@ -0,0 +1 @@
+rootProject.name="testProject"
```

Comment:
Commenting here but is more of a general issue.
Please rename this directory to ` plugin/src/test/resources/` (or ` plugin/src/test/resources/testProject/`).

The reason behind this is that the `test` gradle task has the resources directory as InputDirectory. This means that a change of a file inside this directory (say `build.gradle.kts`) will trigger a cache invalidation and the task will be actually executed. With the current setup, running twice the `test` task will result in the task being skipped the second time, even if you edit the `build.gradle.kts` in between.

Reply:
Good catch, thanks.

Reply:
I actually went a slightly different route and added `inputs.dir("src/test/testProject")` to the "test" task (See https://github.com/martinbonnin/swagger-gradle-codegen/blob/f8385348e283509e8cdc4bc03ab04b651bec857f/plugin/build.gradle.kts#L112)

The rationale is that the `testProject` files don't actually need to be resources, they're never accessed as such so it would be a bit misleading to put them in the resources directory. Let me know if it's ok for you.

Reply:
Yup that works for me